includes/acl/algorithm/uniformly_sampled/decoder.h

```diff
@@ -61,13 +61,14 @@ namespace acl
 
 		namespace impl
 		{
-			// TODO: Ensure RO data is aligned to 64 bytes and RW data is aligned to 64 bytes as well
-			struct DecompressionContext
+			static constexpr size_t CONTEXT_ALIGN_AS = 64;
+
+			struct alignas(CONTEXT_ALIGN_AS) DecompressionContext
 			{
 				// Read-only data
-				const uint32_t* default_tracks_bitset;
```

Comment:
We might also remove the clip argument from the function signatures since it will be within the context. Actually, I am thinking that maybe the context should be a public type instead of hidden under the impl namespace here. Leave it up to the caller to reserve a memory buffer for it and initialize the context within. Create the context from the settings and clip, etc. For now this is fine, but later on once we have key reduction and other algorithms, we'll be able to see how they are really used and refactor this bit out. I'll create a backlog trello card.